src/commands/command-executor.js

```diff
@@ -265,7 +265,8 @@ class CommandExecutor {
             handler.logger.warn(`Command will not be executed.`);
             return;
         }
-        if (name === 'eventListenerCommand') {
+
+        if (['eventListenerCommand', 'shardingTableCheckCommand'].includes(name)) {
```

Comment:
What does this logic do?

Reply:
Executes these default commands without the delay